doc/interop-test-descriptions.md

```diff
@@ -941,6 +962,17 @@ payload body of size `SimpleRequest.response_size` bytes and type as appropriate
 for the `SimpleRequest.response_type`. If the server does not support the
 `response_type`, then it should fail the RPC with `INVALID_ARGUMENT`.
 
+### CacheableUnaryCall
+
+Server gets the default SimpleRequest proto as the request. The content of the
+request are ignored. It returns the SimpleResponse proto with the payload set 
+to current timestamp string.  In addition it adds
```

Comment:
"payload set to the current timestamp string" What is the format and timestamp resolution? I think you could just specify that the format doesn't matter, but you should specify timestamp resolution, because if they used second precision the test would appear to pass commonly even if caching is broken.